content/practices/design-week.md

```diff
@@ -0,0 +1,10 @@
+---
+date: 2018-02-16T00:00:00-05:00
+title: Design Week
+draft: false
+stages: ['delivery']
+---
+
+## What is it?
+
+The first week of the residency. TODO tell me more!
```

Comment:
Should we call this "design week" or "sprint zero" to be in line with the terms that other teams use?

It might also be worth linking to a few resources:

- [The Design Sprint (Google Ventures)](http://www.gv.com/sprint/)
- [What is Sprint Zero?](https://www.scrumalliance.org/community/articles/2013/september/what-is-sprint-zero)

Reply:
In general I'm not a fan of the term sprint, or sprint zero. I like design week.

Reply:
@sherl0cks I also am not a fan of sprint 0 for lots of reasons.

A general andserious question we need to ask ourselves soon is what is our position on Scrum terminology. Lots of companies are choosing more neutral language (e.g. Iterations instead of Sprints, Showcases instead of Sprint Reviews, Daily Standup instead Daily Scrum, Iteration Managers instead of Scrum Masters).

Reply:
https://github.com/rht-labs/practice-library/issues/90